lib/service_actor.rb

```diff
@@ -55,6 +75,7 @@ def initialize(context)
   end
 
   # To implement in your actors.
+  # I like raising NotImplementedError in those cases.
```

Comment:
I would usually do that too. However I ended up not doing that here because of how I decided to use modules that called `super`. I don't think it would make much sense for the *first* module called to NOT call `super` because it is the first, jsut to be able to bypass raising a `NotImplementedError`.